services/movie.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if genres_ids is None and actors_ids is None:
```

Comment:
No need to have this block of code

```
if genres_ids is None and actors_ids is None:
        return Movie.objects.all()
    queryset = Movie.objects.all()
    if genres_ids and actors_ids:
        queryset = queryset.filter(
            genres__id__in=genres_ids
        ).filter(
            actors__id__in=actors_ids
        )
```

Instead just define:
`queryset = Movie.objects.all()`

Reply:
Not resolved